pyspider/database/elasticsearch/projectdb.py

```diff
@@ -57,8 +59,7 @@ def get_all(self, fields=None):
             yield record['_source']
 
     def get(self, name, fields=None):
-        ret = self.es.get(index=self.index, doc_type=self.__type__, id=name,
-                          _source_include=fields or [], ignore=404)
```

Comment:
I can't remember, but doesn't the default behavior for ProjectDB is to return None when a project doesn't exists?